README.md

```diff
@@ -72,3 +72,51 @@ Logging can be controlled using LogObserver provided along or you can use your o
 py.test
 ```
 
+# Security Considerations
+This tool makes use of Python's [`pickle`](https://docs.python.org/2/library/pickle.html) module to receive data from Graphite.
+The `pickle` module is not intended to be secure against maliciously-constructed data.
+In particular, specially-crafted payloads can be used to [execute arbitrary shell commands](https://blog.nelhage.com/2011/03/exploiting-pickle/) on the receiving side.
+For this reason, the forwarder uses a `SafeUnpickler` to restrict what classes can be deserialized, at the cost of speed.
+Normally, this shouldn't be a concern. However, if your application needs to deserialize objects at a faster rate, and the input is already known to be secure, the `get_unpickler` can made to return the default, insecure pickler.
+
+# Failure Considerations
+
+Errors in operation will naturally fall into two realms, input and output.
+Errors on the inbound side will be related to the pickled data coming from carbon.
+Errors on the outbound side will be unexpected (or missing or erroneous) HTTP responses from the blueflood instance to which the metrics are being sent.
+
+## Accepting metrics in pickle format:
+
+If a properly formatted pickle message is truncated during transmission, the metric won't be recorded, and no error or exception will be logged.
+
+If a pickle message is not properly formatted (e.g. if `payload` gets truncated in `sendPickle.py` before `handle` is set), then the unpickler will raise `EOFError` and the exception will be written to the log.
+
+## Forwarding metrics via HTTP:
+
+The forwarder sends metrics to blueflood via an HTTP POST request to the `/v2.0//ingest` [sic] path.
```

Comment:
Possibly, but that's exactly the request uri that I saw in manual testing. It's likely a problem with how the forwarder constructs the destination uri.